lib/streamlit/hello.py

```diff
@@ -180,53 +258,69 @@ def get_UN_data():
         st.error("Connection Error. This demo requires internet access")
         return
 
-    countries = st.multiselect("Choose countries", list(df.index),
-                               ["China", "United States of America"])
+    countries = st.multiselect(
+        "Choose countries", list(df.index), ["China", "United States of America"]
+    )
     if not countries:
         st.error("Please select at least one country.")
         return
 
-    "### Gross Agricultural Production \[$1B\]"
-    st.write(df.loc[countries].sort_index())
+    st.write("### Gross Agricultural Production ($)", df.loc[countries].sort_index())
```

Comment:
To doublecheck the numbers: https://www.statista.com/statistics/196099/total-us-gross-farm-income-since-2000/

I think our numbers are off by X1000.